src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -49718,31 +49801,37 @@ static void Main()
     {
         A a;
         a = new B() { F = new A(), G = new object() };
-        a.F.ToString(); // 1
+        a.F.ToString();
         a = new A();
-        a.F.ToString(); // 2
+        a.F.ToString(); // 1
```

Comment:
>// 1 [](start = 24, length = 4)

Are these trailing comments supposed to correspond to the warnings emitted during compilation? They seem out of sync.